recipes/boost-ext-ut/all/conanfile.py

```diff
@@ -26,7 +26,7 @@ def _minimum_compilers_version(self):
             "Visual Studio": "16",
             "gcc": "9",
             "clang": "9",
-            "apple-clang": "11",
+            "apple-clang": "12",
```

Comment:
I would make this minimum compiler version, `boost-ext-ut` version dependent.
e.g. `11 if tools.Version(self.version) < ... else 12`

Reply:
Should I to that even if we don't package any versions less than 1.1.8? Like this: `11 if tools.Version(self.version) < "1.1.8"  else 12`?

Reply:
Idk, I was under the impression that earlier versions were able to build with apple-clang 11.

Reply:
Done.

Reply:
I bet you're probably right about that. I've added the check.